hudi-common/src/main/java/org/apache/hudi/common/table/timeline/TimelineUtils.java

```diff
@@ -210,11 +210,30 @@ public static HoodieDefaultTimeline getTimeline(HoodieTableMetaClient metaClient
     return activeTimeline;
   }
 
+  /**
+   * Returns a Hudi timeline with commits after the given instant time (exclusive).
+   *
+   * @param metaClient                {@link HoodieTableMetaClient} instance.
+   * @param exclusiveStartInstantTime Start instant time (exclusive).
+   * @return Hudi timeline.
+   */
+  public static HoodieTimeline getCommitsTimelineAfter(
+      HoodieTableMetaClient metaClient, String exclusiveStartInstantTime) {
+    HoodieActiveTimeline activeTimeline = metaClient.getActiveTimeline();
+    HoodieDefaultTimeline timeline =
+        activeTimeline.isBeforeTimelineStarts(exclusiveStartInstantTime)
```

Comment:
@yihua  I have a doubt, since rollback and commit are archived separately, is it possible that there is a very early rollback instant, causing activeTimeline.isBeforeTimelineStarts(exclusiveStartInstantTime) to return false?

Reply:
@BruceKellan You bring up a good point.  So you are talking about the following case where the timeline is like:
```
ts3.rollback, ts50.commit, ts51.commit, ts52.commit, ...
```
and `ts49.commit` and `ts48.commit` are archived.

If we pass in `ts47` as the `exclusiveStartInstantTime` at this point, `activeTimeline.isBeforeTimelineStarts(exclusiveStartInstantTime)` returns `false`.  However, the active timeline misses `ts48.commit` and `ts49.commit` which are required for meta sync.  This is a problem if `ts48.commit` or `ts49.commit` has partition changes.

I'll put up a fix on that.  Thanks for catching it!

Reply:
yes, that's it

Reply:
Here's the fix: #8991